### ISSUE TITLE
Live caption

### ISSUE THREAD

**Issue by Revadike** [brave/brave-browser#15640]
I'd like to see live caption option available (in the accessibility settings):

 * https://www.engadget.com/google-live-caption-chromebooks-095044653.html
 * https://www.msn.com/en-us/news/technology/how-to-set-up-chrome-live-captions-on-your-desktop/ar-BB1eSOUk

**Comment by rebron** [comment]
brave://flags#enable-accessibility-live-caption

We'd need to add back the Accessibility section which was previously just a link out the Chrome Web Store for accessibility extensions.

**Comment by Revadike** [comment]
Yes, I already enabled it there, the option to enable even shows up on the now playing panel, but it doesn't work when enabled. I tested it on chrome and when enabled in accessibility settings, it starts downloading some program data. I think that's what's missing.

**Comment by jmytch** [comment]
Yeh I agree with adding `Live Caption` to Brave. Please seriously add them, because I really want to move away from Chrome over to Brave as my primary browser. 

Any  website that have audio, it will automatically caption it which is good for Hard of Hearing and Deaf out there like myself.

**Comment by Songkeys** [comment]
Any progress so far? This is the only reason I'm still keeping chrome.

**Comment by Revadike** [comment]
Yeah, it's a real shame this is not available on Brave yet

**Comment by jmytch** [comment]
Yeah would be nice if we have update on this please?

As it seems like all Chromium don't have this freely available because Google isn't willing to share? Same for Microsoft Edge, Vivaldi who are using Chromium.

**Comment by TuXEKRO** [comment]
After 10 months, is there any news on whether this is going to be included? The truth is that it would be good for people who do not enjoy a perfect ear

**Comment by UjCbFwtBayFM** [comment]
Any news? People are still asking for this feature.

- https://www.reddit.com/r/brave/comments/vgmmo8/when_will_brave_support_auto_captioning_of_videos/
- https://community.brave.com/t/live-caption-from-chrome/250927/8
- https://community.brave.com/t/accessibility-options/397972
- https://community.brave.com/t/chrome-live-caption/384927
- https://community.brave.com/t/accessibility-section-and-live-caption/169135

**Comment by Songkeys** [comment]
If you are using macOS, the new version Ventura will have a system live caption feature, which will be public in September this year (and you could roll in the beta test if you can't wait). I bet it'll be sooner than Brave supporting this. 🤣

**Comment by OrphanObliterator** [comment]
i would buy brave for money if it had this

**Comment by xplshn** [comment]
4 years later, and Brave still lacks this feature.

**Comment by lynxisdeaf** [comment]
hi im deaf please PLEASE add this feature.

I feel like this would be incredibly easy to implement, and i will literally do it FOR YOU for FREE. I don't care at this point, I just want to stop having to open google chrome for youtube videos and such just for captions.

I shouldn't have to have two browsers installed because one has taken almost 5 YEARS to implement an accessibility feature.

**Comment by tigerza117** [comment]
Interesting
